src/edu/ucsb/cs56/projects/games/rock_paper_scissors/GameGUI.java

```diff
@@ -85,9 +92,9 @@ public void setUpHomeScreen(){
 		//String path3 = "/Users/edwardgonzalez/Desktop/CHOICE/src/rps/images/bulbasaur.jpg";
 
 		try {
-			java.net.URL path1 = new URL("http://cs.ucsb.edu/~gegonzalez/cs56/S12/issues/0000513/browse/src/rps/images/charmander.jpg");
-			java.net.URL path2 = new URL("http://cs.ucsb.edu/~gegonzalez/cs56/S12/issues/0000513/browse/src/rps/images/squirtle.jpg");
-			java.net.URL path3 = new URL("http://cs.ucsb.edu/~gegonzalez/cs56/S12/issues/0000513/browse/src/rps/images/bulbasaur.jpg");
+			java.net.URL path1 = new URL("https://raw.githubusercontent.com/giovannidominguez/cs56-games-rock-paper-scissors/master/src/edu/ucsb/cs56/projects/games/rock_paper_scissors/images/charmander.jpg");
```

Comment:
Great job finding hard coded link to student's files on CSIL. Having a link to a github repo is a better choice; however, there is a better way to do it. Try including any .jpg, .wav, etc. (assets) in the jar file. Always run from the jar, and you will always have the assets. If the internet were to be unavailable, you would not be able to access the images from Github.